src/redux/theme/theme.sagas.js

```diff
@@ -0,0 +1,10 @@
+import { takeEvery } from 'redux-saga/effects';
+import { SET_THEME_MODE } from './theme.types';
+
+function* handleThemeChange({ payload }) {
+  yield localStorage.setItem('darkMode', payload);
```

Comment:
Please, store the theme in the same way as it is done in frontend project where you have would `horondi` object which contains multiple things like jwt token and theme. It is better to follow the same practices for both projects